.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,48 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'java' ]
+
+    steps:
+    - name: Checkout repository
```

Comment:
I spot that indentation here is slightly different comparing to PBS' file

imho it makes sense to make both files as identical as possible

Reply:
never mind, it's my formatter, the indentation is fine